src/content/api/programs.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Programs
+description: Program model and related endpoints
+nav:
+  path: API
+  order: 17
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A Program can be used to provide a versatile foundation for diverse organisational requirements.
+Comprising three distinct types - `loyalty`, `acceptance` and `retail`.
```

Comment:
This is hard to get right because program is a very open concept. I'd suggest something like:
'A program is a container for one or more pieces of user-facing functionality, to be surfaced in both Centrapay and third-party applications.'

I'd remove the descriptions of the different types below as they're not implemented, and just have the supported 'loyalty' type in the Model